### DISCUSSION TITLE
Query Data Tables

### DISCUSSION THREAD

**Discussion by spacec0de** [outline/outline#2700] [Ideas]
Would it be possible to add the ability to construct query data tables (either natively or via API plug-in)? 

The inspiration for this request comes from [Obsidian](https://obsidian.md) and it's powerful [Dataview plug-in](https://blacksmithgu.github.io/obsidian-dataview)

You can pull in data from anywhere in the database and display it in tables, among many other things

**Comment by johnoscott** [comment]
I like this idea very much.

**Comment by ecker00** [comment]
This would be amazing. Sounds also very similar to Notion's linked tables and table views.
